app/views/questionnaire.py

```diff
@@ -266,12 +267,12 @@ def submit_answers(routing_path, eq_id, form_type, collection_id):
     answer_store = get_answer_store(current_user)
 
     if is_completed:
-        message = convert_answers(
+        message = json.dumps(convert_answers(
             metadata,
             g.schema_json,
             answer_store,
             routing_path,
-        )
+        ))
```

Comment:
We need to be happy that passing a JSON string rather than a Python object to encrypt does the same thing. The encrypt method calls the `JWTHelper.encode` method in sdc-cryptography JWTHelper, but that method doesn't have any tests. We should add a test there that tests a dict and a `json.dumps` of that dict produces the same token.

JSON claims as a string is supported by JWCrypto, but there aren't any tests for it. I've raised an issue for this - https://github.com/latchset/jwcrypto/issues/96

Reply:
👍 will add a test this morning